.pre-commit-config.yaml

```diff
@@ -9,6 +9,30 @@ repos:
     hooks:
     - id: isort
 
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.812
+    hooks:
+    - id: mypy
+      args: [
+          --no-strict-optional,
+          --ignore-missing-imports,
+          --follow-imports, skip  # needed so we only test enabled files
+        ]
+      exclude: |
+        (?x)^(
+        fv3core/utils/grid.py |
+        fv3core/utils/gt4py_utils.py |
+        fv3core/decorators.py |
+        fv3core/stencils/fillz.py |
+        fv3core/stencils/moist_cv.py |
+        fv3core/stencils/corners.py |
+        fv3core/stencils/rayleigh_super.py |
+        fv3core/stencils/map_single.py |
+        fv3core/stencils/remap_profile.py |
+        fv3core/stencils/fv_subgridz.py |
+        fv3core/testing/parallel_translate.py
```

Comment:
way less than I expected!

Reply:
Me too! This was a lot longer, but the majority of the files only had the issue that `gtstencil` could not be statically typed.